opentelemetry-appender-tracing/benches/logs.rs

```diff
@@ -34,7 +34,7 @@ struct NoopExporter {
 
 #[async_trait]
 impl LogExporter for NoopExporter {
-    async fn export(&mut self, _: Vec<(&LogRecord, &InstrumentationLibrary)>) -> LogResult<()> {
+    async fn export(&mut self, _: &[(&LogRecord, &InstrumentationLibrary)]) -> LogResult<()> {
```

Comment:
Is it possible spec adding more fields than `LogRecord` and `InstrumentationLibrary`? Having a tuple here means we cannot add another field without a breaking change

Reply:
The specs for logs (including data model) is already stable. And any additive change (if done) would be in Log Data model, which is covered by LogRecord struct.